content/en/user-guide/cloud-pods/auto-load/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Auto-Load"
+weight: 99
+description: >
+  How to load automatically Cloud Pods in LocalStack environment
+aliases:
+  - /user-guide/cloud-pods/auto-load/
```

Comment:
don't think we need the alias since there has never been a page for this before?